examples/calibrateTask.py

```diff
@@ -27,18 +27,24 @@
 import numpy as np
 
 import lsst.utils
-import lsst.pipe.base              as pipeBase
-import lsst.daf.base               as dafBase
-import lsst.afw.coord              as afwCoord
-import lsst.afw.geom               as afwGeom
-import lsst.afw.table              as afwTable
-import lsst.afw.image              as afwImage
-import lsst.afw.display.ds9        as ds9
-from lsst.meas.astrom import AstrometryTask
+import lsst.pipe.base as pipeBase
+from lsst.daf.butlerUtils import ExposureIdInfo
+import lsst.afw.display as afwDisplay
+import lsst.afw.table as afwTable
+import lsst.afw.image as afwImage
+from lsst.pex.config import Config
+from lsst.afw.coord import Coord
+from lsst.afw.geom import PointD
+from lsst.meas.algorithms import LoadReferenceObjectsTask
```

Comment:
Changing these imports has nothing to do with the goal of this ticket or the declared contents of the commits. Making such changes just generates churn (and makes my job as a reviewer harder).  I would like to see this undone.  If it absolutely must be done, then it should be done in a separate commit.